k8s/core-service/core-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: core-deployment
+  labels:
+    app: core
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: core
+  template:
+    metadata:
+      labels:
+        app: core
+    spec:
+      containers:
+        - name: core
+          image: mateusz43211/tourdeshot:core_latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          env:
+            - name: SPRING_DATA_MONGODB_HOST
+              value: mongo
+            - name: SPRING_DATA_MONGODB_PORT
+              value: '27017'
+            - name: SPRING_DATA_MONGODB_DATABASE
+              valueFrom:
+                configMapKeyRef:
+                  name: mongo-core-config
+                  key: MONGODB_TDS_CORE_DBNAME
+            - name: SPRING_DATA_MONGODB_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: mongo-core
+                  key: db-username
+            - name: SPRING_DATA_MONGODB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mongo-core
+                  key: db-password
+            - name: SPRING_DATA_MONGODB_AUTHENTICATION_DATABASE
+              valueFrom:
+                configMapKeyRef:
+                  name: mongo-core-config
+                  key: MONGODB_TDS_CORE_DBNAME
+            - name: SPRING_PROFILES_ACTIVE
+              value: dev
```

Comment:
for now we are deploing with dev profile?

Reply:
With dev profile enabled inital data is loaded. In prod mode there will be no profiles - all config will be loaded from kubernetes